components/dashboard/src/projects/Projects.tsx

```diff
@@ -67,7 +68,7 @@ export default function () {
         <Header title="Projects" subtitle="Manage recently added projects." />
         {projects.length < 1 && (
             <div>
-                <img alt="Projects (empty)" className="h-44 mt-24 mx-auto" role="presentation" src={projectsEmpty} />
+                <img alt="Projects (empty)" className="h-44 mt-24 mx-auto" role="presentation" src={document.documentElement.classList.contains('dark') ? projectsEmptyDark : projectsEmpty} />
```

Comment:
**issue**: The illustration does not get updated when you change theme unless you reload the page. This may only be noticable when you change theme in another window or when system option is set in theme preferences. Could this be a small fix we could add in this PR? 😭 

| Light Theme | Dark Theme |
|-|-|
| <img width="1440" alt="Screenshot 2021-07-28 at 11 53 41 AM" src="https://user-images.githubusercontent.com/120486/127294170-4841e27a-4fde-41d4-aa0f-a33f3c4664a6.png"> | <img width="1440" alt="Screenshot 2021-07-28 at 11 53 53 AM" src="https://user-images.githubusercontent.com/120486/127294174-a4e63473-0e49-47bd-8098-d2138c9d7f59.png"> |

Reply:
Hmm, there are a few places where we adjust the theme based on `document.documentElement.classList.contains('dark')`, but switching in a different tab doesn't cause a re-render.

This could be fixed, but the fix probably isn't small (will probably require a state + an effect)

Reply:
Aha, there is hope: https://caniuse.com/mutationobserver